fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_BrakeTemp.js

```diff
@@ -1,16 +1,159 @@
+// Say when simulation is ready to provide trustworthy ambient temperature - unsure if it works as expected
+function isReady() {
+    return SimVar.GetSimVarValue('L:A32NX_IS_READY', 'number') === 1;
+}
+
+// Return random value between min and max
+function getRandomArbitrary(min, max) {
+    return Math.random() * (max - min) + min;
+}
+
+/**
+ * Scale factor for heat up
+ */
+const HEAT_UP_SCALE = 0.000035;
+
 /**
  * Minimum RPM for counting a wheel set as spinning (brake temperature can increase)
  */
 const WHEEL_RPM_THRESHOLD = 10;
+
 /**
- * Scale factor for heat up
+ * Represents a full landing gear
  */
-const HEAT_UP_SCALE = 0.000035;
+class A32NX_GearStatus {
+    /**
+     *
+     * @param {number} gearIndex Index from left to right, starts at zero
+     * @param {string} gearPositionSimVar SimVar name for this gear's position
+     * @param {string} brakePositionSimVar SimVar name for this gear's brake pedal position
+     * @param {string} wheelRPMSimVar SimVar Name for this gear's wheel RPM
+     */
+    constructor(gearIndex, gearPositionSimVar, brakePositionSimVar, wheelRPMSimVar) {
+        this.gearIndex = gearIndex;
+        this.gearPositionSimVar = gearPositionSimVar;
+        this.gearPosition = 0;
+        this.brakePositionSimVar = brakePositionSimVar;
+        this.brakePosition = 0;
+        this.wheelRPMSimVar = wheelRPMSimVar;
+        this.wheelRPM = 0;
+    }
 
-const BASE_HEAT_UP_FACTOR = 0.003;
+    /**
+     * Update internal state from simulation values
+     * @param {number} deltaTime
+     */
+    update(deltaTime) {
```

Comment:
I think you can just remove this `deltaTime` parameter entirely.

Reply:
One thing I hate about JS/TS is that you can have mismatching signatures like that. 😬

Reply:
Yes I think it's better with the param in order not to forget it's called every frame....